### PR TITLE
installplan for volSync

### PR DESCRIPTION
**Pull request by hhpatel14**
##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by kaovilai** [comment]
Pr to redhat-cop not migtools

**Comment by hhpatel14** [comment]
@kaovilai This is testing env. We need to test this before PR to redhat-cop

**Comment by hhpatel14** [comment]
@kaovilai This is testing env. We need to test this before PR to redhat-cop

File: ansible/roles/ocp4-workload-oadp/tasks/workload.yml [hunk 1]
```diff
@@ -4,22 +4,67 @@
...
+        name: "{{ oadp_namespace }}"
+        kind: Namespace
+        state: "{{ oadp_state }}"
+    - name: "Set openshift-operators namespace"
+      k8s:
+        name: openshift-operators
+        kind: Namespace
+        state: openshift-operators
+
+- name: "Set OperatorGroup and Subscription"
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/tasks/workload.yml]:
```suggestion
- name: "Set OADP OperatorGroup and Subscription"
```

File: ansible/roles/ocp4-workload-oadp/tasks/workload.yml [hunk 2]
```diff
@@ -4,22 +4,67 @@
...
+  - name: "Set OADP OperatorGroup state to {{ oadp_state }}"
+    k8s:
+      state: "{{ oadp_state }}"
+      definition: "{{ lookup('template', 'oadp-operator-operatorgroup.yml.j2' ) }}"
 
-- name: "Set OADP OperatorGroup state to {{ oadp_state }}"
-  k8s:
-    state: "{{ oadp_state }}"
-    definition: "{{ lookup('template', 'oadp-operator-operatorgroup.yml.j2' ) }}"
+  - name: "Set VolSycn OperatorGroup"
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/tasks/workload.yml]:
```suggestion
  - name: "Set VolSync OperatorGroup"
```

File: ansible/roles/ocp4-workload-oadp/tasks/workload.yml [hunk 3]
```diff
@@ -4,22 +4,67 @@
...
+    k8s:
+      state: "{{ oadp_state }}"
+      definition: "{{ lookup('template', 'volSync-operatorgroup.yml.j2' ) }}"
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/tasks/workload.yml]:
```suggestion
      definition: "{{ lookup('template', 'volsync-operatorgroup.yml.j2' ) }}"
```
Let's not mix cases here?

File: ansible/roles/ocp4-workload-oadp/templates/volSync-subscription.yml.j2 [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: volsync-product
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Manual
+  name: volsync-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: volsync-product.v0.5.1-0.1666718165.p
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/templates/volSync-subscription.yml.j2:12]: Does this looks right to you?

Comment (hhpatel14) [reply ansible/roles/ocp4-workload-oadp/templates/volSync-subscription.yml.j2:12]: i installed volsync manually and got the information from there

**Pull request merged**